src/osa/scripts/closer.py

```diff
@@ -526,5 +529,44 @@ def cherenkov_transparency(cmd: List[str]):
         log.debug("Simulate launching scripts")
 
 
+def get_latest_version(longterm_files: List[str]) -> str:
+    """Get the latest version of the produced longterm DL1 datacheck files."""
+    nightdir = date_to_dir(options.date)
+    latest_version = 0
+    for file in longterm_files:
+        idx1 = file.find("/v0.")
+        idx2 = file.find(f"/{nightdir}")
+        version = file[idx1+1:idx2]
+        if int(version[3:])>latest_version:
+            latest_version = int(version[3:])
+            
+    return "v0."+str(latest_version)
+
+
+def create_longterm_symlink():
+    """If the created longterm DL1 datacheck file corresponds to the latest 
+    version available, make symlink to it in the "all" common directory."""
+    nightdir = date_to_dir(options.date)
+    longterm_dir = Path(cfg.get("LST1", "LONGTERM_DIR"))
+    longterm_datacheck_file = longterm_dir / options.prod_id / nightdir / f"DL1_datacheck_{nightdir}.h5"
+    linked_longterm_file = longterm_dir / f"night_wise/all/DL1_datacheck_{nightdir}.h5"
+    all_longterm_files = glob.glob(longterm_dir + f"/v*/{nightdir}/DL1_datacheck_{nightdir}.h5")
```

Comment:
it could also be that given the `all_longterm_files` for that given date, you can get the latest-prod-id one with:
```python
def get_latest_version(longterm_files):
    latest_version_path = max(longterm_files, key=lambda path: path.parent.name.split(".")[1] if path.parent.name.startswith("v") else "")
    return latest_version_path
```
and then always link that version without having to check if there is only 1 longterm file or several. Simply link always the last one? Would that work?

Reply:
yes, I think it works, but I needed to make some changes to that function to work correctlty